client/js/components/day/employee.js

```diff
@@ -17,10 +17,16 @@ var Employee = React.createClass({
 
   // connectDragSource added to props by DragSource
   render: function(){
+    var ratings = ["#F26350", "#F26350", "#6BCFFF", "#6BCFFF", "#6BCFFF", "#A8E5A7"];
```

Comment:
This is a little picky but it would be cool if we could map these hex values to names that reflected what 'rating' the color is representing